fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -173,7 +174,7 @@ public Value createValue(final Node node,
      * @throws RepositoryException
      */
     public Value createValue(final ValueFactory valueFactory, final RDFNode data, final int type)
```

Comment:
I tried adding test but gave up after not finding a way to get hold of the blank nodes created in JcrRdfToolsTest. Are you looking for existence of pairtree mixin on the hierarchy nodes? Thanks.